detection-rules/link_fake_storage_alert.yml

```
@@ -59,7 +59,7 @@ source: |
     or strings.icontains(subject.subject, "exceeded")
     or strings.icontains(subject.subject, "out of")
     or strings.icontains(subject.subject, "icloud")
-    or strings.icontains(subject.subject, "limit")
+    or regex.icontains(subject.subject, '\blimit\b')
```

```suggestion
    or regex.icontains(subject.subject, '\blimit(?:ed|\b)')
```
Should take care of the mimic failures